### PR TITLE
152 cosypose evaluation

### LINKED ISSUE #152: CosyPose evaluation
Minor bugs found when running run_full_cosypose_eval_new.py

### PR DESCRIPTION
**Pull request by MedericFourmy**
- fixes cosypose evaluation minor bugs #152
- adapts eval to [bop_toolkit fork](https://github.com/agimus-project/bop_toolkit) update #153 
- removes [evaluation] optional pip flag

### CODE REVIEW

File: happypose/pose_estimators/cosypose/cosypose/datasets/bop.py [hunk 1]
```diff
@@ -6,15 +6,13 @@
...
 
-from happypose.pose_estimators.cosypose.cosypose.config import BOP_TOOLKIT_DIR, MEMORY
+from happypose.pose_estimators.cosypose.cosypose.config import MEMORY
 from happypose.pose_estimators.cosypose.cosypose.utils.logging import get_logger
 from happypose.toolbox.lib3d.transform import Transform
 
-sys.path.append(str(BOP_TOOLKIT_DIR))
-from bop_toolkit_lib import inout
-
 sys.path = sys.path[:-1]
```
Comment (nim65s) [inline happypose/pose_estimators/cosypose/cosypose/datasets/bop.py]: while here, do you know what is this about ? I'd be glad to remove this monstruosity :)

File: happypose/pose_estimators/cosypose/cosypose/evaluation/evaluation.py [hunk 2]
```diff
@@ -189,22 +185,7 @@ def run_eval(
...
-        mesh_db,
-    ) = happypose.toolbox.inference.utils.load_pose_models(
-        coarse_run_id=cfg.coarse_run_id,
-        refiner_run_id=cfg.refiner_run_id,
-        n_workers=cfg.n_rendering_workers,
-        obj_ds_name=obj_ds_name,
-        urdf_ds_name=urdf_ds_name,
-        force_panda3d_renderer=True,
-    )
-    """
```
Comment (nim65s) [inline happypose/pose_estimators/cosypose/cosypose/evaluation/evaluation.py:207]: cc @ElliotMaitre

**Review by nim65s — APPROVED** [review]
Nice, thanks a lot !

From my point of view, this can be merged as it. Feel free to look at the comment though :)

File: happypose/pose_estimators/cosypose/cosypose/datasets/bop.py (hunk 1 above)
Comment (nim65s) [reply happypose/pose_estimators/cosypose/cosypose/datasets/bop.py]: I'm talking about `sys.path = sys.path[:-1]` : we should avoid all `sys.path` use and manipulation inside the software.

Comment (MedericFourmy) [reply happypose/pose_estimators/cosypose/cosypose/datasets/bop.py]: I agree! Just pushed a commit

Comment (nim65s) [reply happypose/pose_estimators/cosypose/cosypose/datasets/bop.py]: thanks ❤️

**Comment by nim65s** [comment]
Ok, maybe we should also look at the CI fails before merging 😅

**Pull request merged**